src/EnergyPlus/WindTurbine.cc

```diff
@@ -688,46 +688,48 @@ namespace WindTurbine {
-                                        "Generator Turbine Power Coefficient",
-                                        OutputProcessor::Unit::None,
-                                        state.dataWindTurbine->WindTurbineSys(WindTurbineNum).PowerCoeff,
-                                        OutputProcessor::SOVTimeStepType::System,
-                                        OutputProcessor::SOVStoreType::Average,
-                                        state.dataWindTurbine->WindTurbineSys(WindTurbineNum).Name);
-                } else if (SELECT_CASE_var == RotorType::VAWT) {
-                    SetupOutputVariable(state,
-                                        "Generator Turbine Chordal Component Velocity",
-                                        OutputProcessor::Unit::m_s,
-                                        state.dataWindTurbine->WindTurbineSys(WindTurbineNum).ChordalVel,
-                                        OutputProcessor::SOVTimeStepType::System,
-                                        OutputProcessor::SOVStoreType::Average,
-                                        state.dataWindTurbine->WindTurbineSys(WindTurbineNum).Name);
-                    SetupOutputVariable(state,
-                                        "Generator Turbine Normal Component Velocity",
-                                        OutputProcessor::Unit::m_s,
-                                        state.dataWindTurbine->WindTurbineSys(WindTurbineNum).NormalVel,
-                                        OutputProcessor::SOVTimeStepType::System,
-                                        OutputProcessor::SOVStoreType::Average,
-                                        state.dataWindTurbine->WindTurbineSys(WindTurbineNum).Name);
-                    SetupOutputVariable(state,
-                                        "Generator Turbine Relative Flow Velocity",
-                                        OutputProcessor::Unit::m_s,
-                                        state.dataWindTurbine->WindTurbineSys(WindTurbineNum).RelFlowVel,
-                                        OutputProcessor::SOVTimeStepType::System,
-                                        OutputProcessor::SOVStoreType::Average,
-                                        state.dataWindTurbine->WindTurbineSys(WindTurbineNum).Name);
-                    SetupOutputVariable(state,
-                                        "Generator Turbine Attack Angle",
-                                        OutputProcessor::Unit::deg,
-                                        state.dataWindTurbine->WindTurbineSys(WindTurbineNum).AngOfAttack,
-                                        OutputProcessor::SOVTimeStepType::System,
-                                        OutputProcessor::SOVStoreType::Average,
-                                        state.dataWindTurbine->WindTurbineSys(WindTurbineNum).Name);
-                }
+            switch (state.dataWindTurbine->WindTurbineSys(WindTurbineNum).rotorType) {
```

Comment:
Can start deploying the following pattern here. 

```cpp
auto &windTurbineSys = state.dataWindTurbineSys->WindTurbineSys(WindTurbineNum);
swtich (windTurbineSys.rotorType) {
...
```

Something to start getting used to.